jupytercalpoly/Jupyter-Bifrost
# Added brushing functionality

https://user-images.githubusercontent.com/33008436/126231777-5f167c75-f873-4f4f-8b22-7e006e01611e.mov

**Brush filtering on the graph!**
Fixes #33 
## Changes
- Brushing listener is not debounced to prevent overly active signal update (adaptable).
- `VegaPandasTranslator` now supports multiple types of compound filter queries: `or`, `and`, and `not`.
- Category filtering screen checkboxes now initialize to the state of the spec (correctly checked or unchecked)
- Updated `updateSpecFilter` to handle multiple compound operators.

## Testing
- Open a scatterplot.
- Drag over a region to create a bounding box.
- Click "Apply changes"
- Try to apply other filters on the filter screens and check that nothing breaks

- Open a bar chart.
- Repeat previous tests.

### Notes
- We could definitely combine the brush with the axis based sliders to make it easier to adjust selections precisely.
